### PR TITLE
i18n(es): Update `solid-js`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updates `solid-js` based on these commits 9ecadb8fe8d7a309ca8d06ceae2dcaf578feaf02, 18a21132075abd00923325d38b6857076524d577, 29885a48868fe92c970cfc789167951c686f572b, 27cd3bb82e4543844703d456a89852d09b703390 & 58e1da0415dca383b7f528b47f24e5658ef50317

#### Related issues & labels (optional)

- Closes N/A 
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/solid-js.mdx [hunk 1]
```diff
@@ -102,22 +128,52 @@ export default defineConfig({
...
+// CharacterName.tsx
+function CharacterName() {
+  const [name] = createResource(() =>
+    fetch('https://swapi.dev/api/people/1')
+      .then((result) => result.json())
+      .then((data) => data.name)
+  );
+  return (
+    <>
+      <h2>Name:</h2>
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/solid-js.mdx:156]:
```suggestion
      <h2>Nombre:</h2>
```

File: src/content/docs/es/guides/integrations-guide/solid-js.mdx [hunk 2]
```diff
@@ -102,22 +128,52 @@ export default defineConfig({
...
+      {/* Luke Skywalker */}
+      <div>{name()}</div>
+    </>
+  );
+}
+```
 
-## Contribuyendo
+Similarmente, los [componentes Lazy][solid-lazy-components] de Solid también se resolverán y su HTML se incluirá en la página inicial renderizada en el servidor.
+Los componentes [`client:only`][astro-client-only] que no se hidratan no se envuelven automáticamente en límites de suspenso.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/solid-js.mdx:165]:
```suggestion
Los componentes [`client:only`][astro-client-only] que no se hidratan no se envuelven automáticamente en límites de Suspense.
```

File: src/content/docs/es/guides/integrations-guide/solid-js.mdx [hunk 3]
```diff
@@ -102,22 +128,52 @@ export default defineConfig({
...
 
-Este paquete es mantenido por el equipo central de Astro. ¡Estás invitado a enviar un problema o PR!
+Sientete libre de agregar límites de suspenso adicionales según tu preferencia.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/solid-js.mdx:167]:
```suggestion
Sientete libre de agregar límites de Suspense adicionales según tu preferencia.
```

File: src/content/docs/es/guides/integrations-guide/solid-js.mdx [hunk 4]
```diff
@@ -102,22 +128,52 @@ export default defineConfig({
...
-## Solución de problemas
-
-Para obtener ayuda, consulta el canal de `#support` en [Discord](https://astro.build/chat). ¡Nuestros amables miembros del equipo de soporte están aquí para ayudar!
-
-También puedes revisar nuestra [documentación de integraciónes de Astro][astro-integration] para obtener más información sobre las integraciones.
+## Uso
+
+Usa un componente SolidJS como lo harías con cualquier [componente de framework UI](/es/core-concepts/framework-components/).
+
+### Límites de suspenso
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/solid-js.mdx:141]:
```suggestion
### Límites de Suspense
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Pull request merged**